components/partials/JobOffersWidgetItem.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <div class="job-offers-widget-item widget-item">
+    <span :class="['job-offers-widget-item__technology-icon', devIconClass]" :aria-hidden="true"></span>
```

Comment:
This icon font shouldn't be hidden for screen readers. Add span with `.visualy-hidden` class contains devIconClass computed property.